pkg/ccn-lite/Makefile

```diff
@@ -7,6 +7,9 @@ include $(RIOTBASE)/pkg/pkg.mk
 
 .PHONY: ..cmake_version_supported
 
+CCNLITE_SOURCE_DIR = $(PKG_SOURCE_DIR)
```

Comment:
aren't these variables (here and the other cmake pkgs) unneeded anymore? Is there a reason to keep them?

Reply:
They are not really needed indeed, we could just use PKG_SOURCE_DIR/PKG_BUILD_DIR with the same result. I'll adapt.